packages/indexer-native/native/src/lib.rs

```diff
@@ -0,0 +1,99 @@
+use arc_swap::ArcSwap;
+use keccak_hash::keccak;
+use lazy_static::lazy_static;
+use native_utils::{
+    errors::{SafeResult, Terminal},
+    marshalling::Arg,
+    proxy::{Proxy, ProxyTerminal},
+    task::run_async,
+};
+use neon::prelude::*;
+use never::Never;
+use secp256k1::{recovery::RecoverableSignature, Message, PublicKey, Secp256k1, VerifyOnly};
+use std::sync::Arc;
+mod attestation;
+mod signature_verification;
+
+use attestation::AttestationSigner;
+use primitive_types::U256;
+use signature_verification::SignatureVerifier;
+use std::convert::TryInto as _;
+type Address = [u8; 20];
```

Comment:
nit: Could use an empty line above to separate it from the imports.

Reply:
Done